bundle-workflow/src/test_workflow/utils/build_manifest_provider.py

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+import os
+
+from aws.s3_bucket import S3Bucket
+from manifests.build_manifest import BuildManifest
+
+
+class BuildManifestProvider:
+    @staticmethod
+    def get_build_manifest_relative_location(build_id, opensearch_version, architecture):
```

Comment:
i'm open to moving these methods to build_manifest.py if that makes more sense

Reply:
Yes it does. This is not Java :)

Reply:
+1 I think it makes it more cleaner to put these utility functions with BundleManifest and BuildManifest.

Reply:
sure, thx for the inputs